lms/djangoapps/discussion_api/api.py

```diff
@@ -515,6 +516,7 @@ def _do_extra_actions(api_content, cc_content, request_fields, actions_form, con
                 else:
                     context["cc_requester"].unvote(cc_content)
                     api_content["vote_count"] -= 1
+                track_voted_event(request, context["course"], cc_content, "up", False if form_value else True)
```

Comment:
Lots of opaque args. Use kwargs instead.  `request` and `context` are self-documenting, but `"up"` and the boolean are particularly hard to understand, and `cc_content` only slightly easier.